File: cs50x/pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    // get positive number
    int number;
    do
    {
        float change = get_float("Change owed: ");
        number = round(change * 100);
    }
    while (number < 1);

    int coins = 0;
    do
    {
        if (number >= 25)
        {
            number-=25;
            coins++;
        }
        else if (number >= 10)
        {
            number-=10;
            coins++;
        }
        else if (number >= 5)
        {
            number-=5;
            coins++;
        }
        else
        {
            number-=1;
            coins++;
        }
    }
    while (number > 0);
    printf("%i\n", coins);
}

File: cs50x/pset6/cash/cash.py
from cs50 import get_float

# getting change
while True:
    change = get_float("Change owed: ")
    if change > 0:
        break

# calculating coins
change = change * 100
coins = int(change / 25)
coins += int(change % 25 / 10)
coins += int(change % 25 % 10 / 5)
coins += int(change % 25 % 10 % 5)
print(coins)